packages/@tinacms/cli/src/buildTina/index.ts

```diff
@@ -0,0 +1,208 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import retry from 'async-retry'
+
+import { Bridge, buildSchema, createDatabase, Database } from '@tinacms/graphql'
+import {
+  FilesystemBridge,
+  FilesystemStore,
+  IsomorphicBridge,
+  LevelStore,
+  Store,
+} from '@tinacms/datalayer'
+import path from 'path'
+
+import { compileSchema, resetGeneratedFolder } from '../cmds/compile'
+import { genClient, genTypes } from '../cmds/query-gen'
+import { makeIsomorphicOptions } from './git'
+import type { GraphQLSchema } from 'graphql'
+
+interface BuildOptions {
+  ctx: any
+  database: Database
+  store: Store
+  bridge: Bridge
+  noWatch?: boolean
+  isomorphicGitBridge?: boolean
+  verbose?: boolean
+  dev?: boolean
+  local?: boolean
+  noSDK?: boolean
+  beforeBuild?: () => Promise<any>
+  afterBuild?: () => Promise<any>
+  skipIndex?: boolean
+}
+
+interface BuildSetupOptions {
+  isomorphicGitBridge?: boolean
+  experimentalData?: boolean
+}
+
+export const buildSetupCmdBuild = async (
+  ctx: any,
+  next: () => void,
+  opts: BuildSetupOptions
+) => {
+  const rootPath = ctx.rootPath as string
+  const { bridge, database, store } = await buildSetup({
+    ...opts,
+    rootPath,
+    useMemoryStore: true,
+  })
+  // attach to context
+  ctx.bridge = bridge
+  ctx.database = database
+  ctx.store = store
+
+  next()
+}
+
+export const buildSetupCmdServerStart = async (
+  ctx: any,
+  next: () => void,
+  opts: BuildSetupOptions
+) => {
+  const rootPath = ctx.rootPath as string
+  const { bridge, database, store } = await buildSetup({
+    ...opts,
+    rootPath,
+    useMemoryStore: false,
+  })
+  // attach to context
+  ctx.bridge = bridge
+  ctx.database = database
+  ctx.store = store
+
+  next()
+}
+
+const buildSetup = async ({
+  isomorphicGitBridge,
+  experimentalData,
+  rootPath,
+  useMemoryStore,
+}: BuildSetupOptions & { rootPath: string; useMemoryStore: boolean }) => {
+  const fsBridge = new FilesystemBridge(rootPath)
+  const isomorphicOptions =
+    isomorphicGitBridge && (await makeIsomorphicOptions(fsBridge))
+
+  /**
+   * To work with Github directly, replace the Bridge and Store
+   * and ensure you've provided your access token.
+   * NOTE: when talking the the tinacms repo, you must
+   * give your personal access token access to the TinaCMS org
+   */
+  // const ghConfig = {
+  //   rootPath: 'examples/tina-cloud-starter',
+  //   accessToken: '<my-token>',
+  //   owner: 'tinacms',
+  //   repo: 'tinacms',
+  //   ref: 'add-data-store',
+  // }
+  // const bridge = new GithubBridge(ghConfig)
+  // const store = new GithubStore(ghConfig)
+
+  const bridge = isomorphicGitBridge
+    ? new IsomorphicBridge(rootPath, isomorphicOptions)
+    : fsBridge
+
+  // const store = experimentalData
+  //   ? new LevelStore(rootPath, useMemoryStore)
+  //   : new FilesystemStore({ rootPath })
+
+  const store = new LevelStore(rootPath, useMemoryStore)
```

Comment:
@logan-anderson I think it should be ok. Essentially that is going to be saying that --experimentalData is always enabled (in local mode)

Reply:
Will this result in the `experimentalData` getting set to `true` in that case?

Reply:
I checked and it looks like the value in the _schema.json is coming directly from the flag, so no, I think it won't

Reply:
Also, I'm wondering with this change if we should be removing the experimentalData cli flag altogether in this PR?

Reply:
@kldavis4 it does not  do anything anymore so we don't need to have it.

It might be annoying to remove it all together (this would cause the CLI to crash if someone does pass it) but we can make it a warning for now and remove it at 1.0?

Reply:
yeah, emitting a warning makes sense (along with removing experimentalData anywhere else)